convert.py

```diff
@@ -160,13 +161,14 @@ def guessed(model: 'LazyModel') -> 'Params':
     def loadHFTransformerJson(model: 'LazyModel', config_path: 'Path') -> 'Params':
         config = json.load(open(config_path))
 
-        n_vocab    = config["vocab_size"]
-        n_embd     = config["hidden_size"]
-        n_layer    = config["num_hidden_layers"]
-        n_ff       = config["intermediate_size"]
-        n_head     = config["num_attention_heads"]
-        n_head_kv  = config["num_key_value_heads"] if "num_key_value_heads" in config else n_head
-        f_norm_eps = config["rms_norm_eps"]
+        n_vocab      = config["vocab_size"]
+        n_embd       = config["hidden_size"]
+        n_layer      = config["num_hidden_layers"]
+        n_ff         = config["intermediate_size"]
+        n_head       = config["num_attention_heads"]
+        n_head_kv    = config["num_key_value_heads"] if "num_key_value_heads" in config else n_head
+        f_norm_eps   = config["rms_norm_eps"]
+        f_rope_scale = config.get("rope_scaling", {}).get("factor", None) if config.get("rope_scaling", {}).get("type", "") == "linear" else None
```

Comment:
nit: dict.get returns None as a fallback by default. This would be more readable:
```suggestion
        rope_scaling = config.get("rope_scaling", {})
        f_rope_scale = rope_scaling.get("factor") if rope_scaling.get("type") == "linear" else None
```

Reply:
I would be happy with even a couple of plain `if` statements. But yeah, that one-liner a bit hard to read, anything that improves the readability would be good.

Reply:
I was overfitting a bit to the other implementations trying to match the style. I am not a fan of tertiary operators or long lines, so I would also prefer an `if/else` implementation (tested and committed now).